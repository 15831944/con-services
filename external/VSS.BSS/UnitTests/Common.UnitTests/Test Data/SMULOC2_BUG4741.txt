<?xml version="1.0" encoding="utf-8"?>
<smuLoc:topic xmlns:smuLoc="http://www.cat.com/em/feed/v2/ServiceMeterLocation"
xmlns:feed="http://www.cat.com/em/feed/v2/Common"
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:schemaLocation="http://www.cat.com/em/feed/v2/ServiceMeterLocation https://emfeed.cat.com/EMFeedFiles/schema/v2/ServiceMeterLocation.xsd">

  <nextBuffer>
    <url>
    https://emfeed.cat.com/EMFeedService/v2/feed/nighthawk/SMULOC/23595024</url>
    <moreData>true</moreData>
  </nextBuffer>
  <message>
    <header>
      <messageId>23592919</messageId>
      <masterMsgId>30803980</masterMsgId>
      <equipment>
        <serialNumber>3YK00343</serialNumber>
        <make>CAT</make>
        <model>135H</model>
        <nickname>16001</nickname>
      </equipment>
      <moduleCode>0IBDJ231513M1</moduleCode>
      <moduleTime>2009-09-26 05:35:00</moduleTime>
      <receivedTime>2009-09-26 05:45:14</receivedTime>
    </header>
    <serviceMeter>
      <value>4174</value>
      <uom>Hours</uom>
    </serviceMeter>
    <location>
      <latitude>33.473705</latitude>
      <longitude>-81.982056</longitude>
      <validity>V</validity>
    </location>
  </message>
</smuLoc:topic>
