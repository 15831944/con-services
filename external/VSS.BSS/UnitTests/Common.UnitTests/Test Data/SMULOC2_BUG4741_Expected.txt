<?xml version="1.0" encoding="utf-8"?>
<ArrayOfInboundDeviceEvent xmlns:i="http://www.w3.org/2001/XMLSchema-instance"
xmlns="http://www.nighthawk.com/nighthawk/data/rptevent/2009/03">
  <InboundDeviceEvent i:type="InboundDeviceEventPosition">
    <Address i:nil="true" />
    <Asset>
      <AssetID>2866</AssetID>
      <CustomerID>233</CustomerID>
    </Asset>
    <Device>
      <DeviceID i:nil="true" />
      <DeviceTypeID>9</DeviceTypeID>
      <ExtDeviceID>0IBDJ231513M1</ExtDeviceID>
    </Device>
    <EventTypeID>1</EventTypeID>
    <EventUTC>2009-09-26T05:35:00</EventUTC>
    <ExtDriverID i:nil="true" />
    <ExternalCorrelationID>30803980</ExternalCorrelationID>
    <ExternalMessageID>23592919</ExternalMessageID>
    <ExternalSourceID>1</ExternalSourceID>
    <Position>
      <ElevationFeet i:nil="true" />
      <Heading i:nil="true" />
      <Latitude>33.473705</Latitude>
      <Longitude>-81.982056</Longitude>
      <Mileage i:nil="true" />
      <SpeedMph i:nil="true" />
    </Position>
    <ReceivedUTC>2009-09-26T05:45:14</ReceivedUTC>
    <Site i:nil="true" />
    <TriggerType i:nil="true" />
  </InboundDeviceEvent>
  <InboundDeviceEvent i:type="InboundDeviceEventUtilization">
    <Address i:nil="true" />
    <Asset>
      <AssetID>2866</AssetID>
      <CustomerID>233</CustomerID>
    </Asset>
    <Device>
      <DeviceID i:nil="true" />
      <DeviceTypeID>9</DeviceTypeID>
      <ExtDeviceID>0IBDJ231513M1</ExtDeviceID>
    </Device>
    <EventTypeID>73</EventTypeID>
    <EventUTC>2009-09-26T05:35:00</EventUTC>
    <ExtDriverID i:nil="true" />
    <ExternalCorrelationID>30803980</ExternalCorrelationID>
    <ExternalMessageID>23592919</ExternalMessageID>
    <ExternalSourceID>1</ExternalSourceID>
    <Position>
      <ElevationFeet i:nil="true" />
      <Heading i:nil="true" />
      <Latitude>33.473705</Latitude>
      <Longitude>-81.982056</Longitude>
      <Mileage i:nil="true" />
      <SpeedMph i:nil="true" />
    </Position>
    <ReceivedUTC>2009-09-26T05:45:14</ReceivedUTC>
    <Site i:nil="true" />
    <TriggerType i:nil="true" />
    <IsOn>true</IsOn>
    <MaxSpeed i:nil="true" />
    <RuntimeSeconds>15026400</RuntimeSeconds>
    <RuntimeType>WorkingIdleCum</RuntimeType>
    <Sensor>
      <Description i:nil="true" />
      <SensorIsOn i:nil="true" />
      <SensorName i:nil="true" />
      <SensorNumber i:nil="true" />
    </Sensor>
    <StartStopUTC i:nil="true" />
  </InboundDeviceEvent>
</ArrayOfInboundDeviceEvent>
